.github/workflows/positron-merge-to-main.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           POSITRON_PY_VER_SEL: 3.10.12
           POSITRON_R_VER_SEL: 4.4.0
         id: electron-smoke-tests
-        run: DISPLAY=:10 yarn ${{ env.SMOKETEST_TARGET }} --tracing --parallel --jobs 2 --skip-cleanup
```

Comment:
have there been problems with 2 workers? i haven't seen any

Reply:
oh, i thought you mentioned yesterday that PR runs were more unreliable than full test runs?  maybe I was not following you.